suite/fuzz/fuzz_diff.c

```diff
@@ -285,7 +285,8 @@ int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
                 snprintf(CapstoneAssemblyText, 80, "\t%s\t%s", insn->mnemonic, insn->op_str);
             }
             normalize_hex(CapstoneAssemblyText);
-            if (strcmp(CapstoneAssemblyText, LLVMAssemblyText) != 0) {
+            //allow llvm output to be shorter for shl edi != shl edi, 1
+            if (strncmp(CapstoneAssemblyText, LLVMAssemblyText, strlen(LLVMAssemblyText)) != 0) {
```

Comment:
is "shorter" a good fix? how about another approach, so we whitelist all the special cases like this?

Reply:
This is indeed not good. It was not supposed to be a fix but just a test.
I begin to doubt that we can have something relevant without whitelisting half the opcodes.

I just quickly got this other inconsistency :
capstone add	al, 255 != llvm 	add	al, -1
`./cstool -d x64 "0x04 0xFF"`
`echo -n -e "0x04 0xff" | ./bin/llvm-mc -disassemble -output-asm-variant=1 -print-imm-hex -triple=x86_64`

Maybe I should look for another tool than llvm-mc, maybe objdump.
What do you think ?

Reply:
yes, objdump is another good choice. we can support them all.